create_sdf_from_mesh.py

```diff
@@ -0,0 +1,441 @@
+"""
```

Comment:
can we put this into the manipulation package (e.g. the manipulation subdirectory)?  that way it will be available via pip, on deepnote, etc.

Reply:
done